pq-crypto/kyber_r3/symmetric-shake.c

```diff
@@ -0,0 +1,63 @@
+#include "fips202.h"
+#include "symmetric.h"
+#include <stdlib.h>
+
```

Comment:
I'd trust your judgment over mine here, but I just want to check - do we not need to `#include <stddef.h>
#include <stdint.h>` because of the use of `size_t` and `uint8_t` etc.?

Reply:
`stddef.h` and `stdint.h` get included transitively. We include `fips202.h` here, and in `fips202.h` we have:

```
#include <stddef.h>
#include <stdint.h>
```

If we missing a required header for our types, the CI builds would certainly let us know ;)